interfaces/devdoc/threadpool_requirements.md

```diff
@@ -78,29 +71,23 @@ MOCKABLE_FUNCTION(, void, threadpool_destroy, THREADPOOL_HANDLE, threadpool);
 
 **NON_THREADPOOL_01_007: [** `threadpool_destroy` shall perform an implicit close if `threadpool` is OPEN. **]**
 
-### threadpool_open_async
+### threadpool_open
 
 ```c
-MOCKABLE_FUNCTION(, int, threadpool_open_async, THREADPOOL_HANDLE, threadpool, ON_THREADPOOL_OPEN_COMPLETE, on_open_complete, void*, on_open_complete_context);
+MOCKABLE_FUNCTION(, int, threadpool_open, THREADPOOL_HANDLE, threadpool);
 ```
 
-`threadpool_open_async` opens the threadpool object so that subsequent calls to `threadpool_schedule_work` can be made.
-
-**NON_THREADPOOL_01_008: [** If `threadpool` is `NULL`, `threadpool_open_async` shall fail and return a non-zero value. **]**
-
-**NON_THREADPOOL_01_009: [** If `on_open_complete` is `NULL`, `threadpool_open_async` shall fail and return a non-zero value. **]**
-
-**NON_THREADPOOL_01_010: [** `on_open_complete_context` shall be allowed to be `NULL`. **]**
+`threadpool_open` opens the threadpool object so that subsequent calls to `threadpool_schedule_work` can be made.
 
-**NON_THREADPOOL_01_011: [** Otherwise, `threadpool_open_async` shall switch the state to OPENING and perform any actions needed to open the `threadpool` object. **]**
+**NON_THREADPOOL_01_008: [** If `threadpool` is `NULL`, `threadpool_open` shall fail and return a non-zero value. **]**
 
-**NON_THREADPOOL_01_012: [** On success, `threadpool_open_async` shall return 0. **]**
+**NON_THREADPOOL_01_009: [** If `on_open_complete` is `NULL`, `threadpool_open` shall fail and return a non-zero value. **]**
```

Comment:
> If `on_open_complete` is `NULL [](http://example.com/codeflow?start=28&length=31)

remove

Reply:
Why do we have 3 md's for threadpool?

Reply:
The PAL is organized such that there is an MD for the interface, with only high level requirements (likely could be tested in the int test), then an MD for each platform with actual requirements like shall call x-API (which are definitely meant to be tested in platform-specific UT's).

I think it makes sense to have the separate docs, though maybe repeating the header isn't needed in 3 places...